web/cat/routes/upload.py

```diff
@@ -42,9 +36,7 @@ async def rabbithole_upload_endpoint(
         )
 
     # upload file to long term memory, in the background
-    background_tasks.add_task(
-        ingest_file, ccat, file, chunk_size=chunk_size, chunk_overlap=chunk_overlap
-    )
+    background_tasks.add_task(ccat.send_file_in_rabbit_hole, file)
```

Comment:
the function call should take `chunk_size` and `chunk_overlap` args as it was before

Reply:
@nicola-corbellini to try out the endpoints see `localhost:1865/docs/`

Reply:
Ouch sorry, I naively misunderstood the code. 
I'll find a way to fix this, as before removing the `chunk_size` and `chunk_overlap` params from the endpoint it raised an error. I'll dig into this!